app/Http/Controllers/MembershipController.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use HMS\Entities\Role;
```

Comment:
As an aside the number of includes here tells me this is quite the fat controller. Might be worth figuring out how we can offload some of this stuff to some service classes.